lib/isURL.js

```diff
@@ -103,6 +103,9 @@ function isURL(url, options) {
 
   split = url.split('@');
   if (split.length > 1) {
+    if (options.exclude_auth_url) {
```

Comment:
Can you change this to `disallow_auth` for consistency with the other options?

Reply:
Yes can do.